models/agent_model.go

```diff
@@ -165,8 +166,12 @@ func (s *Agent) DSN(service *Service, dialTimeout time.Duration, database string
 		cfg := mysql.NewConfig()
 		cfg.User = username
 		cfg.Passwd = password
-		cfg.Net = "tcp"
-		cfg.Addr = net.JoinHostPort(host, strconv.Itoa(int(port)))
+		cfg.Net = "unix"
+		cfg.Addr = socket
+		if cfg.Addr == "" {
```

Comment:
maybe it's better to rewrite it like `if socket == "" {`
It'll improve readability